src/Tribe/API.php

```diff
@@ -93,11 +93,11 @@ public static function saveEventMeta( $event_id, $data, $event = null ) {
 			$data = self::prepare_event_date_meta( $event_id, $data );
 
 			if ( empty( $data['EventHideFromUpcoming'] ) ) {
-				delete_post_meta( $event_id, '_EventHideFromUpcoming' );
+				delete_metadata('post', $event_id, '_EventHideFromUpcoming' );
```

Comment:
On this line and all tweaked `*_metadata` lines, can you add a space between `(` and `'`.